.gitlab-ci.yml

```diff
@@ -64,3 +64,23 @@ build:
     - docker push $AWS_REGISTRY/kilt/prototype-chain:$CI_COMMIT_SHORT_SHA
     - docker push $AWS_REGISTRY/kilt-parachain/collator:latest-develop
     - docker push $AWS_REGISTRY/kilt-parachain/collator:$CI_COMMIT_SHORT_SHA
+
+build-wasm:
+  image:
+    name: paritytech/srtool:1.53.0
+    entrypoint: [""]
+  stage: build
+  only:
+    - develop
+    - master
+  script:
+    - export PACKAGE=peregrine-runtime
```

Comment:
I suppose we also want the `spiritnet-runtime`?

Reply:
fixed in d104b68